nipype/interfaces/freesurfer/longitudinal.py

```diff
@@ -251,3 +253,207 @@ def _list_outputs(self):
         outputs = self.output_spec().get()
         outputs["out_file"] = os.path.abspath(self.inputs.out_file)
         return outputs
+
+
+class BaseReconAllInputSpec(ReconAllInputSpec):
+    subject_id = traits.Str(argstr="-subjid %s", desc="subject name")
```

Comment:
My understanding is that `subject_id` should not be specified when calling `recon-all -base` and it's the base template ID which supersedes it.

```suggestion
```